composer.json

```diff
@@ -1,91 +1,90 @@
 {
-    "name": "acmephp/acmephp",
```

Comment:
You should only add the new package line: updating this whole file makes reviewing difficult